src/handlers/assign.rs

```diff
@@ -665,13 +727,23 @@ impl fmt::Display for FindReviewerError {
                 write!(
                     f,
                     "Could not assign reviewer from: `{}`.\n\
-                     User(s) `{}` are either the PR author, already assigned, or on vacation, \
-                     and there are no other candidates.\n\
-                     Use `r?` to specify someone else to assign.",
+                     User(s) `{}` are either the PR author, already assigned, or on vacation. \
+                     If it's a team, we could not find any candidates.\n\
```

Comment:
Not needed, but we could just be smart and use a message tailored to if it was a team or not.

Reply:
Now that I read this message again, I think mentioning a team there doesn't make a lot of sense. The "team not found" reason should be already covered by the enum item `FindReviewerError::TeamNotFound`.

Think I'll just remove that sentence.